tests/lib.rs

```diff
@@ -50,6 +64,32 @@ fn test_error_response_handler() {
     assert_eq!(res.unwrap_err(), "Error: 400 Bad Request. Cannot retrieve the user count unless the channel is a presence channel")
 }
 
+#[test]
+fn test_cluster_builder() {
+    let client = hyper::Client::with_connector(ClusterRequest::default());
+    let mut pusher = Pusher::new("1", "2", "3").client(client).cluster("eu").finalize();
+
+    let res = pusher.channel_users("presence-yolo");
+    let users = res.unwrap().users;
+    let user_one = &users[0];
+    let user_two = &users[1];
+    assert_eq!(user_one.id, "red");
+    assert_eq!(user_two.id, "blue")
+}
+
+#[test]
+fn test_secure() {
+    let client = hyper::Client::with_connector(SecureRequest::default());
+    let mut pusher = Pusher::new("1", "2", "3").client(client).secure().host("127.0.0.1").finalize();
+
+    let res = pusher.channel_users("presence-yolo");
+    let users = res.unwrap().users;
+    let user_one = &users[0];
+    let user_two = &users[1];
+    assert_eq!(user_one.id, "red");
+    assert_eq!(user_two.id, "blue")
+}
+
```

Comment:
Could you add another tests which sets both `host` and `cluster` to show which overwrites the other.